tests/integration/pilot/common/apps.go

```diff
@@ -274,58 +314,30 @@ func SetupApps(t resource.Context, i istio.Instance, apps *EchoDeployments) erro
 	apps.StatefulSet = echos.Match(echo.Service(StatefulSetSvc))
 	apps.Naked = echos.Match(echo.Service(NakedSvc))
 	apps.External = echos.Match(echo.Service(ExternalSvc))
-	apps.ProxylessGRPC = echos.Match(echo.Service(ProxylessGRPCSvc))
+	if !t.Clusters().IsMulticluster() {
```

Comment:
ProxylessGRPCSvc is only created when t.Clusters().IsMulticluster() is false
istio/tests/integration/pilot/common/apps.go